src/java/voldemort/server/VoldemortConfig.java

```diff
@@ -3231,4 +3241,37 @@ public String getFileFetcherClass() {
         return this.fileFetcherClass;
     }
 
+    public String getRdbDataDirectory() {
+        return rdbDataDirectory;
+    }
+
+    /**
+     * Where RocksDB should put its data directories
+     * 
+     * <ul>
+     * <li>Property :"rocksdb.data.dir"</li>
+     * <li>Default : "/tmp/rdb_data_dir"</li>
+     * </ul>
+     * 
+     * @param rdbDataDirectory
+     */
+    public void setRdbDataDirectory(String rdbDataDirectory) {
+        this.rdbDataDirectory = rdbDataDirectory;
+    }
+
+    public boolean getRocksdbPrefixKeysWithPartitionId() {
+        return rocksdbPrefixKeysWithPartitionId;
+    }
+
+    /**
+     * If true, keys will be prefixed by the partition Id on disk. This can
+     * possibly speed up rebalancing, restore operations, at the cost of 2 bytes
+     * of extra storage per key
+     * 
+     * @param rocksdbPrefixKeysWithPartitionId
+     */
+    public void setRocksdbPrefixKeysWithPartitionId(boolean rocksdbPrefixKeysWithPartitionId) {
```

Comment:
so it does not make much sense for this flag to even exist now.. (its abundantly clear that the extra 2 bytes in the key is totally worth it, for all the speedup we get with admin ops).. So may be have one server level config to enable it for all store types on the same server?